pkg/daemon/ceph/client/pool.go

```diff
@@ -393,3 +404,14 @@ func GetPoolStatistics(context *clusterd.Context, name, namespace string) (*Pool
 
 	return &poolStats, nil
 }
+
+// IsPropertyInSpec verifies whether a property already exist in the list
+func IsPropertyInSpec(properties []cephv1.PoolProperty, property string) bool {
```

Comment:
If this is only called by the pool controller, how about moving it to that package?

Reply:
Gone since PoolProperties is a `map[string]string` now.